packages/client/tests/functional/metrics/enabled/tests.ts

```diff
@@ -6,39 +6,499 @@ import type { PrismaClient } from './node_modules/@prisma/client'
 declare let prisma: PrismaClient
 declare let newPrismaClient: NewPrismaClient<typeof PrismaClient>
 
+const executeOneQuery = async () => {
+  const email = 'user@example.com'
+  await prisma.user.create({
+    data: {
+      email,
+    },
+  })
+  await prisma.user.findFirst({ where: { email } })
+}
+
 testMatrix.setupTestSuite(
   ({ provider }) => {
     describe('empty', () => {
-      test('does not crash before client is connected', async () => {
+      test('$metrics.prometheus() does not crash before client is connected', async () => {
         await expect(prisma.$metrics.prometheus()).resolves.not.toThrow()
       })
+      test('$metrics.json() does not crash before client is connected', async () => {
+        await expect(prisma.$metrics.json()).resolves.not.toThrow()
+      })
     })
 
-    describe('with data', () => {
-      beforeAll(async () => {
-        const email = 'user@example.com'
-        await prisma.user.create({
-          data: {
-            email,
-          },
+    describe('before a query', () => {
+      test('MongoDB: should have the same keys, before and after a query', async () => {
+        if (provider !== 'mongodb') {
+          return
+        }
+
+        const metricBefore = await prisma.$metrics.json()
+        // console.log(JSON.stringify(metricBefore, null, 2))
+        expect(metricBefore).toMatchObject({
+          counters: [
+            {
+              key: 'prisma_client_queries_total',
+              labels: {},
+              value: 0,
+              description: 'Total number of Prisma Client queries executed',
+            },
+            {
+              key: 'prisma_datasource_queries_total',
+              labels: {},
+              value: 0,
+              description: 'Total number of Datasource Queries executed',
+            },
+            {
+              key: 'prisma_pool_connections_closed_total',
+              labels: {},
+              value: 0,
+              description: '',
+              // description: 'Total number of Pool Connections closed',
```

Comment:
New finding while doing a snapshot (ping @janpio)

MongoDB somehow is missing some descriptions for
- prisma_pool_connections_closed_total
- prisma_pool_connections_opened_total
- prisma_client_queries_active
- prisma_pool_connections_busy
- prisma_pool_connections_idle
- prisma_pool_connections_open

Reply:
There is a PR for reworking descriptions: https://github.com/prisma/prisma-engines/pull/4239